app/models.py

```diff
@@ -87,6 +87,7 @@ class Member(db.Model):
     self_sufficiency_matrices = db.relationship('Member_Self_Sufficiency_Matrix', backref='member', lazy=True)
     self_efficacy_quizzes = db.relationship('Member_Self_Efficacy_Quiz', backref='member', lazy=True)
     user = db.relationship("User", back_populates="member", lazy=True)
+    club = db.relationship("Club", back_populates="members", lazy=True)
```

Comment:
Shouldn't this back_pop "member" instead of "members"?

Reply:
The relationship in the Club table is defined as 'members' while the one in the User table is defined as 'member' therefore it is members here